build/lib/src/asset/id.dart

```diff
@@ -110,9 +110,9 @@ class AssetId implements Comparable<AssetId> {
   /// Note that this is intended for communicating ids across isolates and not
   /// for persistent storage of asset identifiers. There is no guarantee of
   /// backwards compatibility in serialization form across versions.
-  AssetId.deserialize(data)
-      : package = data[0],
-        path = data[1];
+  AssetId.deserialize(List data)
```

Comment:
Can we do `List<String>` to be more specific?

Also we should update the comment as it is not applicable to package:build (I think it was copied from barback)